src/components/display/Dropdown.test.tsx

```diff
@@ -144,52 +129,47 @@ describe('Dropdown', () => {
 	test('dropdown inside a modal open and close properly', async () => {
 		const onClose = jest.fn();
 		const onClick = jest.fn();
-		render(
+		const { user } = setup(
 			<Modal open title="modal with dropdown" onClose={onClose}>
 				<Dropdown items={items}>
 					<Button label="opener" onClick={onClick} />
 				</Dropdown>
 			</Modal>
 		);
-
+		// run timers of modal
+		act(() => jest.runOnlyPendingTimers());
```

Comment:
this line should not be needed anymore since we are waiting for the text now with the findBy